.gitignore

```diff
@@ -1,5 +1,6 @@
 # Compiled class file
 *.class
+test.md
```

Comment:
- [ ] It is important to set `node_modules/` directory into your `.gitignore` file. This directory can be added to your repository by accident while setting up your linters, so this is a good practice to avoid further problems.

Reply:
Thanks for the review. But I don't know what is "node_modules/"

Reply:
It's the forder that is generated when you run `npm install`

Reply:
I still don't understand, but I will do a recherche on it thanks.